nestkernel/ring_buffer.h

```diff
@@ -351,7 +352,30 @@ ListRingBuffer::get_index_( const delay d ) const
   assert( ( size_t ) idx < buffer_.size() );
   return idx;
 }
-}
+
+
+template < unsigned int num_values >
+class MultiValueRingBuffer
+{
+public:
+  MultiValueRingBuffer();
+
+  void add_value( const long idx0, const long idx1, const double val );
```

Comment:
`idx0` and `idx1` are little informative. How about `time_idx` and `channel_idx` or similar?

Reply:
They are now called `slot` and `channel`, respectively. Okay?